Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj

```diff
@@ -97,7 +97,7 @@
     <Compile Include="Tiled\TmxTerrain.cs" />
     <Compile Include="Tiled\TmxTile.cs" />
     <Compile Include="Tiled\TmxTileOffset.cs" />
-    <Compile Include="Tiled\TmxTileset.cs" />
+    <Compile Include="Tiled\TmxTileSet.cs" />
```

Comment:
It's okay I've already done it. I don't have a Linux machine to test on though please make sure it works.